src/plugins/irc-events/link.js

```diff
@@ -65,6 +66,7 @@ function parse(msg, preview, res, client) {
 			|| $('meta[name="twitter:image:src"]').attr("content")
 			|| $('link[rel="image_src"]').attr("href")
 			|| "";
+		preview.type = res.type;
```

Comment:
unnecessary?

Reply:
yeah, but may as well populate if it's there?

Reply:
It's already set to `link` above.

Reply:
fixed